src/components/Menu.js

```diff
@@ -72,6 +72,50 @@ export default class Menu extends Component {
+                          <option value="White Almond Cake">White Almond Cake</option>
+                          <option value="Salted Caramel Cake">Salted Caramel Cake</option>
+                          <option value="Lemon Poppyseed Cake">Lemon Poppyseed Cake</option>
+                        </select>
+                        <label for="icings" className="Cake-Icing-Order">Choose an Icing:</label>
+                        <select name="icings" id="icings">
+                        <option value="---">---</option>
+                          <option value="Vanilla Buttercream">Vanilla Buttercream</option>
+                          <option value="Chocolate Buttercream">Chocolate Buttercream</option>
+                          <option value="Mocha Buttercream">Mocha Buttercream</option>
+                          <option value="Citrus Buttercream">Citrus Buttercream</option>
+                          <option value="Mint Chocolate Chip Buttercream">Mint Chocolate Chip Buttercream</option>
+                          <option value="Maple Buttercream">Maple Buttercream</option>
+                          <option value="Caramel Buttercream">Caramel Buttercream</option>
+                          <option value="Red Velvet Buttercream">Red Velvet Buttercream</option>
+                          <option value="Cream Cheese Frosting">Cream Cheese Frosting</option>
+                          <option value="Lemon Cream Cheese Frosting">Lemon Cream Cheese Frosting</option>
+                        </select>
+                        <label for="fillings" className="Cake-Filling-Order">Choose a Filling:</label>
+                        <select name="fillings" id="fillings">
+                          <option value="---">---</option>
+                          <option value="Raspberry Jam Filling">Raspberry Jam Filling</option>
+                          <option value="Strawberry Jam Filling">Strawberry Jam Filling</option>
+                          <option value="Blackberry Jam Filling">Blackberry Jam Filling</option>
+                          <option value="Dark Chocolate Ganache">Dark Chocolate Ganache</option>
+                        </select>
+                        <label for="Pickup" className="Cake-Pickup">Pickup/Delivery</label>
+                        <select name="Pickup" id="Pickup">
+                        <option value="---">---</option>
+                        <option value="Pickup">Pickup</option>
+                        <option value="Delivery">Delivery</option>
+                        </select>
+                        <label for="Custom" className="Cake-Custom">Customized/Specialty Decorations</label>
+                        <input type="checkbox" id="Custom" name="Custom" value="Custom" className="Custom-Checkbox"/>
+                        <button type="submit" className="Cake-Order-Submit">Submit Cake</button>
+                      </form>
```

Comment:
Can these adopt the same airy feel with whitespace as the rest of the site? Also, please force labels to be inline with their selection objects.